pandas/core/frame.py

```diff
@@ -90,7 +90,7 @@
 from pandas.core.index import Index, ensure_index, ensure_index_from_sequences
 from pandas.core.indexes import base as ibase
 from pandas.core.indexes.datetimes import DatetimeIndex
-from pandas.core.indexes.multi import maybe_droplevels
+from pandas.core.indexes.multi import maybe_droplevels, MultiIndex
```

Comment:
`MultiIndex` should go before `maybe_droplevels`

Reply:
Done.